x/slashing/simulation/operations.go

```diff
@@ -93,14 +93,11 @@ func SimulateMsgUnjail(
 		if err != nil {
 			return simtypes.NoOpMsg(types.ModuleName, msgType, "unable to get validator consensus key"), nil, err
 		}
-		info, err := k.ValidatorSigningInfo.Get(ctx, consAddr)
-		if err != nil {
-			return simtypes.NoOpMsg(types.ModuleName, msgType, "unable to find validator signing info"), nil, err // skip
-		}
+		info, _ := k.ValidatorSigningInfo.Get(ctx, consAddr)
 
-		selfDel, err := sk.Delegation(ctx, simAccount.Address, bz)
+		selfDel, _ := sk.Delegation(ctx, simAccount.Address, bz)
 		if err != nil {
```

Comment:
which error is checked here?

Reply:
Good eye. I have removed the if condition